### PR TITLE
Add checks for searches

### PR DESCRIPTION
**Pull request by xavdid**
Ensure that they return an array (of any length). Relates to https://github.com/zapier/zapier/issues/11924. 

Once merged, I'll do a `patch` release (this shouldn't break existing code, because before today searches that returned objects would have seen an error on the server) and bump cli.

### CODE REVIEW

File: test/checks.js [hunk 1]
```diff
@@ -2,14 +2,19 @@
 
 require('should');
 
-const createIsSingle = require('../src/checks/create-is-single');
+const checks = require('../src/checks');
 
 describe('checks', () => {
```
Comment (BrunoBernardino) [inline test/checks.js]: If we could add some tests for the other checks (even if simple), it'd be 💯

**Review by BrunoBernardino — APPROVED** [review]
This looks great, David! I've got a wishlist item, but it's not required 😄

**Comment by xavdid** [comment]
@BrunoBernardino added tests for all checks! Actually found a bug, so it was probably worth the time. 😁 

I'll merge and deploy on monday!

**Comment by BrunoBernardino** [comment]
Awesome stuff!

**Review by bcooksey — APPROVED** [review]
Solid test coverage, and nice refactor of importing on the checks

**Pull request merged**